client/a8c-for-agencies/components/site-configurations-modal/style.scss

```diff
@@ -17,6 +17,10 @@
 		color: var(--color-neutral-50);
 		font-size: rem(14px);
 		line-height: 1.5;
+
+		.disabled-label {
+			color: var(--studio-gray-5);
```

Comment:
I know we haven't had a design review yet, however I think we could consider using a slightly darker shade of gray, for instance:

```suggestion
			color: var(--studio-gray-20);
```

I found `--studio-gray-20` to be used in other parts of the codebase for disabled elements.

| studio-gray-5 | studio-gray-20 |
|--------|--------|
| ![Markup on 2024-08-02 at 13:27:00](https://github.com/user-attachments/assets/03e18844-b2eb-461a-8055-fb3c05e69dc9) | ![Markup on 2024-08-02 at 13:29:15](https://github.com/user-attachments/assets/953a5085-f1f8-44f5-83b5-bfc423688818) |

Reply:
Yeah, we can still then adjust it after hearing feedback from design.